src/test/tasks/composable-tasks.yaml

```diff
@@ -0,0 +1,96 @@
+tasks:
```

Comment:
I see in the tests how we're checking for the various success vs error outputs. I think these tests would give me more confidence for future extensions if they checked for the value of the input being echo'd instead of just "success"